### ISSUE TITLE
give users accounts

### ISSUE THREAD

**Issue by my0sin** [acmumn/newsite#2]
Allow users to request accounts and password changes through portal

**Comment by mzhang28** [comment]
is this specifically for server accounts? accounts on the website don't have passwords

**Comment by my0sin** [comment]
> is this specifically for server accounts? accounts on the website don't have passwords

yes. maybe have a way to display the available resources to members

**Comment by mzhang28** [comment]
ok then this is probably a more long-term thing
